frontend/src/pages/ErrorsV2/ErrorsV2.tsx

```diff
@@ -143,7 +183,23 @@ export default function ErrorsV2() {
 
 			{!isBlocked && (
 				<Box cssClass={styles.searchPanelContainer}>
-					<SearchPanel />
+					<SearchPanel
+						query={query}
+						setQuery={setQuery}
+						page={page}
+						setPage={setPage}
+						loading={getErrorsData.loading}
+						errorGroups={getErrorsData.errorGroups}
+						moreErrors={getErrorsData.moreErrors}
+						totalCount={getErrorsData.totalCount}
+						histogramBucketSize={getErrorsData.histogramBucketSize}
+						resetMoreErrors={getErrorsData.resetMoreErrors}
+						updateSearchTime={updateSearchTime}
+						rebaseSearchTime={rebaseSearchTime}
+						startDate={startDate}
+						endDate={endDate}
+						selectedPreset={selectedPreset}
+					/>
```

Comment:
Ideally, I'd like to simplify this, but the was losing the context when trying to grab them for the hooks.